### PR TITLE
added atoplex kit on genomes.config

### PR DESCRIPTION
**Pull request by ctuni**
---
name: Added atoplex kit to the genomes.config file for viral recon.
about: See viralrecon [issue](https://github.com/nf-core/viralrecon/issues/353)
---

Please follow these steps before submitting your PR:

- [ ] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any

### CODE REVIEW

**Comment by ljmesi** [comment]
It looks good! I didn't manage to access file: `https://github.com/nf-core/test-datasets/raw/viralrecon/genome/NC_045512.2/amplicon/nCoV-2019.atoplex.V1.bed`. The other ones were accessible.

File: conf/pipeline/viralrecon/genomes.config [hunk 1]
```diff
@@ -90,6 +90,12 @@ params {
             scheme     = 'nCoV-2019'
           }
         }
+        'atoplex' {
+            fasta      = 'https://github.com/nf-core/test-datasets/raw/viralrecon/genome/MN908947.3/primer_schemes/artic/nCoV-2019/V1200/nCoV-2019.reference.fasta'
+            gff        = 'https://github.com/nf-core/test-datasets/raw/viralrecon/genome/NC_045512.2/GCF_009858895.2_ASM985889v3_genomic.200409.gff.gz'
+            primer_bed = 'https://github.com/nf-core/test-datasets/raw/viralrecon/genome/NC_045512.2/amplicon/nCoV-2019.atoplex.V1.bed'
```
Comment (maxulysse) [inline conf/pipeline/viralrecon/genomes.config:96]: Can't access this file

Comment (ctuni) [reply conf/pipeline/viralrecon/genomes.config:96]: Appologies, I will fix it ASAP

Comment (maxulysse) [reply conf/pipeline/viralrecon/genomes.config:96]: Ping me when you're good to go, and I'll have another look

Comment (maxulysse) [reply conf/pipeline/viralrecon/genomes.config:96]: ok, so the issue was that there was a space in the filename, this is now fixed

**Review by maxulysse — APPROVED** [review]

File: conf/pipeline/viralrecon/genomes.config (hunk 1 above)
Comment (ctuni) [reply conf/pipeline/viralrecon/genomes.config:96]: Appologies agai, I'm having a hectic summer :pray: Thank you very much for the fix!!!!

**Pull request merged**